src/test/java/uk/gov/ons/fsdr/tests/acceptance/steps/CommonSteps.java

```diff
@@ -147,15 +147,15 @@ public void theEmployeeIsSentToAllDownstreamServices(String id) throws Exception
     gatewayEventMonitor.grabEventsTriggered("SENDING_XMA_ACTION_RESPONSE", 6, 5000L);
     assertTrue(gatewayEventMonitor.hasEventTriggered(id, "SENDING_XMA_ACTION_RESPONSE", 5000L));
     fsdrUtils.ingestGranby();
-    fsdrUtils.rcaExtract();
+//    fsdrUtils.rcaExtract();
```

Comment:
we can just delete the RCA references in the tests now rather than comment them out